<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PelcoD</class>
 <widget class="QFrame" name="PelcoD">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>265</width>
    <height>363</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::StyledPanel</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Raised</enum>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Pelco D</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <item row="4" column="0">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Данные 1</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Команда 1</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Адрес</string>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>Данные 2</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Команда 2</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="HexSpinBox" name="sb_cmd2"/>
        </item>
        <item row="4" column="1">
         <widget class="HexSpinBox" name="sb_data1"/>
        </item>
        <item row="5" column="1">
         <widget class="HexSpinBox" name="sb_data2"/>
        </item>
        <item row="0" column="1">
         <widget class="HexSpinBox" name="sb_sync"/>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Смнхронизация</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="HexSpinBox" name="sb_address"/>
        </item>
        <item row="2" column="1">
         <widget class="HexSpinBox" name="sb_cmd1"/>
        </item>
        <item row="6" column="0">
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>Контр. сумма</string>
          </property>
         </widget>
        </item>
        <item row="6" column="1">
         <widget class="HexSpinBox" name="sb_chk_sum"/>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="btn_send">
        <property name="text">
         <string>Send</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Presets</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>Адрес</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="sb_addr_cam_presets"/>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="btn_move_position">
        <property name="text">
         <string>move position</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_run_tur_1">
        <property name="text">
         <string>run tur 1</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_wiper">
        <property name="text">
         <string>wiper</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>HexSpinBox</class>
   <extends>QSpinBox</extends>
   <header>hexspinbox.hpp</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>sb_sync</tabstop>
  <tabstop>sb_address</tabstop>
  <tabstop>sb_cmd1</tabstop>
  <tabstop>sb_cmd2</tabstop>
  <tabstop>sb_data1</tabstop>
  <tabstop>sb_data2</tabstop>
  <tabstop>sb_chk_sum</tabstop>
  <tabstop>btn_send</tabstop>
  <tabstop>sb_addr_cam_presets</tabstop>
  <tabstop>btn_move_position</tabstop>
  <tabstop>btn_run_tur_1</tabstop>
  <tabstop>btn_wiper</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
